react/AutocompleteBlock.tsx

```diff
@@ -115,9 +116,17 @@ const AutocompleteBlock: StorefrontFunctionComponent<any &
 
   const { selectedItem, quantitySelected, unitMultiplier } = state
   const callAddToCart = async (items: any) => {
+      let currentItemsInCart = orderForm.orderForm.items
     const mutationResult = await addToCart({
       variables: {
         items: items.map((item: any) => {
+            let existsInCurrentOrder = currentItemsInCart.filter(
+              el => el.id === item.id.toString()
+            )
+            if (existsInCurrentOrder.length > 0) {
+              item['quantity'] =
+                item['quantity'] + existsInCurrentOrder[0]['quantity']
```

Comment:
As it is Destructured after your changes, there's no need to reference it to a position `0`, do                 `item.quantity + existsInCurrentOrder.quantity` instead